scripts/local_testnet_pos/beacon_node.sh

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env bash
+
+#
+# Starts a beacon node based upon a genesis state created by `./setup.sh`.
+#
+
+set -Eeuo pipefail
+
+source ./vars.env
+
+SUBSCRIBE_ALL_SUBNETS="--subscribe-all-subnets"
+DEBUG_LEVEL=${DEBUG_LEVEL:-info}
+
+# Get options
+while getopts "d:sh" flag; do
+  case "${flag}" in
+    d) DEBUG_LEVEL=${OPTARG};;
+    s) SUBSCRIBE_ALL_SUBNETS="--subscribe-all-subnets";;
+    h)
+       echo "Start a beacon node"
+       echo
+       echo "usage: $0 <Options> <DATADIR> <NETWORK-PORT> <HTTP-PORT> <ETH1-PORT> <EL-PORT>"
+       echo
+       echo "Options:"
+       echo "   -s: pass --subscribe-all-subnets to 'lighthouse bn ...', default is not passed"
+       echo "   -d: DEBUG_LEVEL, default info"
+       echo "   -h: this help"
+       echo
+       echo "Positional arguments:"
+       echo "  DATADIR       Value for --datadir parameter"
+       echo "  NETWORK-PORT  Value for --enr-udp-port, --enr-tcp-port and --port"
+       echo "  HTTP-PORT     Value for --http-port"
+       exit
+       ;;
+  esac
+done
+
+# Get positional arguments
+data_dir=${@:$OPTIND+0:1}
+network_port=${@:$OPTIND+1:1}
+http_port=${@:$OPTIND+2:1}
+eth1_port=${@:$OPTIND+3:1}
+el_port=${@:$OPTIND+4:1}
+
+exec lighthouse \
+	--debug-level $DEBUG_LEVEL \
+	bn \
+	$SUBSCRIBE_ALL_SUBNETS \
+	--datadir $data_dir \
+	--testnet-dir $TESTNET_DIR \
+	--enable-private-discovery \
+	--staking \
+	--enr-address 127.0.0.1 \
+	--enr-udp-port $network_port \
+	--enr-tcp-port $network_port \
+	--port $network_port \
+	--http-port $http_port \
+	--disable-packet-filter \
+	--target-peers $((BN_COUNT - 1)) \
+  --eth1 \
+  --merge \
+  --terminal-total-difficulty-override=60000000 \
```

Comment:
It would be great to be able to set the TTD from the `vars.env`, or in lieu of that to have a slightly lower value so that the merge happens sooner. I played around with TTD 5000000 and the merge seemed to happen almost immediately, maybe twice that by default, i.e. 10000000?